.github/CODEOWNERS

```diff
@@ -2,12 +2,12 @@
 # More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 # Default owner
-* @hashicorp/team-ip-compliance
+* @hashicorp/team-ip-compliance @hashicorp/raft-force
```

Comment:
@hashicorp/raft-force also needs write access to this repo.

Reply:
GitHub is showing the following error:

> CODEOWNERS errors
>
> Unknown owner on line 5: make sure the team @hashicorp/raft-force exists, is publicly visible, and has write access to the repository
>
> …@hashicorp/team-ip-compliance @hashicorp/raft-force

Reply:
Oops yeah, I called that out in the other PRs I opened (like https://github.com/hashicorp/go-immutable-radix/pull/61) as @hashicorp/raft-force also needs write access to those repos too. LMK internally if that's a process I should initiate or if IP Compliance can take care of it when reviewing/merging these PRs.

Reply:
IP Compliance can take care of it when the review/merge the PR 👍